### DISCUSSION TITLE
Importing places

### DISCUSSION THREAD

**Discussion by chilburn06** [organicmaps/organicmaps#9346] [Q&A]
I am trying to import my places from google maps, and it keeps telling me I need a file manager but I have 2. Any suggestions?![Screenshot_20240921-104254.png](https://github.com/user-attachments/assets/e1960c0f-83fa-4f33-b3f4-82422164979e)

**Comment by deshudiosh** [comment]
I have the same issue

**Comment by CMSplumbear** [comment]
Same issue here.
Version 2024.08.16-5-FDroid is working perfectly.
Using Obtainium I installed the Github version 2024.09.08-7-Web in parallel, installed a few maps, then attempted to import the bookmarks exported from the F-Droid version and got the message above. 

The Samsung file manager application doesn't propose Organic Maps (nor any other map app)  to "share" or "open with" the exported .kmz file.
Using Ghost Commander it was possible to "open with" as "???" (unknown filetype?) and select Organic Maps to import the file correctly. 
Still, the import dialog doesn't work 
Device is Samsung A50 Android 11

**Comment by hb0nd** [comment]
moved to https://github.com/organicmaps/organicmaps/issues/9390, please follow there
